.github/workflows/test-python.yml

```diff
@@ -133,7 +133,7 @@ jobs:
         shell: bash
         run: |
           maturin build
-          pip install target/wheels/polars-*.whl
```

Comment:
Actually, why don't we use `maturin develop` here?

Reply:
Somehow that didn't work nicely on Windows.. I can revisit, maybe it works normally now. That would simplify the workflow a lot. I'll check and open a new PR if we can substitute `maturin develop`.